Automattic/wp-calypso
# Add oembed provider link for invites section

#### Changes proposed in this Pull Request

* Adds an ombed provider `<link>` tag to the document `<head>` for the invites section. This is to support link preview cards we have added in D73584-code for P2's invite links. 
* For this to work, we've split the section into main (`index.node.js`) vs browser (`index.web.js`) targets, with `index.node.js` middleware restricted to the bare minimum.

#### Testing instructions



1. Send your test user an invite via `/people/new/<site-slug>`. Open the invite link in a new browser tab and replace the domain with the `calypso.live` one, or `calypso.localhost:3000` if testing locally.
2. View page source, and find "oembed". You should see the injected oembed provider links.
3. Regression test: accepting user invites should be unaffected. Send your test user a user invite, and verify that you can accept it without problems.
